README.md

```
@@ -2,313 +2,1430 @@
+- [Bhushan Borole](https://bhushan-borole.github.io)
+- [Binay Shaw](https://binay-shaw.onrender.com) [Mobile Developer]
+- [Binyam Seyoum](https://binyam.io) [Devops & Cloud Solutions Engineer]
+- [Bipin M V](https://bipinmv.netlify.app)
+- [Biplob Sutradhar](https://biplobsd.github.io) [Software Engineer | Android | Web | Chrome
+  Extension]
+- [Bishnu Pokhrel](https://bishnupokhrel.netlify.app) [Full Stack Laravel Vue Developer]
+- [Bjorn Melin](https://bjornmelin.io) [Data Scientist]
+- [Blanc John Clayton](https://www.johnclaytonblanc.com)
+- [Bluezly](https://www.bluezly.exid.me)
+- [Bob Matyas](https://www.bobmatyas.com)
+- [Boddapu Lokesh](https://boddapulokesh.github.io/portfolio) [Ai-Developer][India]
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "india"

(MD052, reference-links-images)